pkg/container-hook/tracer.go

```diff
@@ -57,6 +56,7 @@ import (
 	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets"
 	"github.com/inspektor-gadget/inspektor-gadget/pkg/kfilefields"
 	"github.com/inspektor-gadget/inspektor-gadget/pkg/utils/host"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/utils/notify"
```

Comment:
What about moving `pkg/utils/notify/notify.go` to `pkg/container-hook/runtime-finder/finder.go`?

"utils" is a bit of an anti-pattern: https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common

And when we get rid of `runcfanotify`, it will already be in the right place, in a subdirectory of `container-hook`.

Reply:
IMO it's ok for `runcfanotify` to use a pkg in a subdirectory of `container-hook`.